recipe/meta.yaml

```diff
@@ -30,9 +28,7 @@ requirements:
     - python
     - python-dateutil >=2.6
     - python-tzdata >=2020.1
-    - backports.zoneinfo >=0.2.1  # [py<39]
```

Comment:
This might be necessary for backporting

Reply:
We're not building this for python 3.9 and lower anymore